lucille-core/src/test/java/com/kmwllc/lucille/indexer/OpenSearchIndexerTest.java

```diff
@@ -290,8 +293,12 @@ public void testRouting() throws Exception {
     Map<String, Object> map = (Map<String, Object>) indexRequest.document();
 
     assertEquals("doc1", indexRequest.id());
+
+    // routing has been set appropriately even though routing field has been deleted by ignoreFields
+    // note that id is still in document even though it was in ignoreFields
+    // as the id is later added back to the document
```

Comment:
when interacting with a live opensearch index, does the id show up as a document field in the search engine itself, or are we only seeing the id in the return value of indexRequest.document() in this particular testing context?

Reply:
When documents are indexed into live opensearch index, both id & _id (same value) still shows up as a document field in the search engine itself.